docs/future-work/cmp-hierarchy-improvements/add-associative-connections.md

```diff
@@ -1,4 +1,9 @@
 ---
 title: Add Associative Connections
```

Comment:
Ah ok that makes sense, I was confused by the potential duplication (I think because I focused on the term "hierarchy" in the `cmp-hierarchy` grouping).

With that cleared up, I wonder if it's a bit of a duplication of "Generalize Voting to Associative Connections" --> my temptation would be to keep that one, and add the point that this should enable associating e.g. sound objects with physical objects (i.e. where their models may not both be 3D), and get rid of "Add Associative Connections" under cmp-hierarchy. What do you think @vkakerbeck ?

Reply:
wow yes! That only now clicked for me that those two are basically the same. Its kind of cool that we can solve both these with the same solution. I think I had added this one under hierarchy because the first time I thought about these was in the context of modeling language and grounding it in physical models of objects. But I think we should just remove this one and expand on the one under voting like you suggest. Maybe add the "abstract" or "num_steps" label to it